src/sentry/tagstore/snuba/backend.py

```diff
@@ -486,10 +486,20 @@ def get_tag_value_paginator_for_projects(self, projects, environments, key, star
         snuba_key = snuba.get_snuba_column_name(key)
 
         conditions = []
-        if query:
-            conditions.append([snuba_key, 'LIKE', u'%{}%'.format(query)])
+
+        if snuba_key == 'project_id':
+            if query:
+                try:
+                    conditions.append([snuba_key, '=', int(query)])
+                except ValueError:
+                    pass
```

Comment:
1. Should we even allow project_id conditions here? given that project_id is already added separately as a filter. 
2. If so, and the project_id query param is an invalid int, should we return nothing instead of everything?

Reply:
Or raise, yeah

Reply:
i also had the same thought about whether or not we should allow project_id to be searched for in this endpoint/tagstore method. it's included in this list https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L29 i think because discover uses it. since either way, we have to do some sort of special casing (either here or in the api to say that `project.id` isn't supported or something) it seemed like we might as well support it. but i don't really have strong opinions -- i just wanted to make it not blow up if someone does search for it